docs/reveal-config.js

```diff
@@ -1,73 +1,72 @@
-    height: 1080,
-  
-    theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
-    transition: Reveal.getQueryHash().transition || 'none', // default/cube/page/concave/zoom/linear/fade/none
-  
-    keyboard: {
-      32: function() {
-        var video = document.querySelector('.present video');
-        if (video.paused == true) {
-          video.play();
-        } else {
-          video.pause();
-        }
+  controls: true,
+  progress: true,
+  history: true,
+  center: false,
+  width: 1920,
+  height: 1080,
+
+  theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
+  transition: Reveal.getQueryHash().transition || 'none', // default/cube/page/concave/zoom/linear/fade/none
+
+  keyboard: {
+    32: function () {
+      var video = document.querySelector('.present video');
+      if (video.paused == true) {
+        video.play();
+      } else {
+        video.pause();
       }
     },
-  
-    slideNumber: 'c/t',
-    showSlideNumber: 'speaker',
-  
-    // Optional libraries used to extend on reveal.js
-    dependencies: [
-      {
-        src: './libs/revealjs/js/classList.js',
-        condition: function() {
-          return !document.body.classList;
-        }
+  },
+
+  slideNumber: 'c/t',
+  showSlideNumber: 'speaker',
+
+  // Optional libraries used to extend on reveal.js
+  dependencies: [
+    {
+      src: './libs/revealjs/js/classList.js',
+      condition: function () {
+        return !document.body.classList;
       },
-      {
-        src: './libs/revealjs/plugin/markdown/marked.js',
-        condition: function() {
-          return !!document.querySelector('[data-markdown]');
-        }
+    },
+    {
+      src: './libs/revealjs/plugin/markdown/marked.js',
+      condition: function () {
+        return !!document.querySelector('[data-markdown]');
+      },
+    },
+    {
+      src: './libs/revealjs/plugin/markdown/markdown.js',
+      condition: function () {
+        return !!document.querySelector('[data-markdown]');
       },
-      {
-        src: './libs/revealjs/plugin/markdown/markdown.js',
-        condition: function() {
-          return !!document.querySelector('[data-markdown]');
-        }
+    },
+    {
+      src: './libs/revealjs/plugin/highlight/highlight.js',
+      async: true,
+      callback: function () {
+        hljs.initHighlightingOnLoad();
       },
-      {
-        src: './libs/revealjs/plugin/highlight/highlight.js',
-        async: true,
-        callback: function() {
-          hljs.initHighlightingOnLoad();
-        }
+    },
+    {
+      src: './libs/revealjs/plugin/zoom-js/zoom.js',
+      async: true,
+      condition: function () {
+        return !!document.body.classList;
       },
-      {
-        src: './libs/revealjs/plugin/zoom-js/zoom.js',
-        async: true,
-        condition: function() {
-          return !!document.body.classList;
-        }
+    },
+    {
+      src: 'sfeir-school-theme/js/reveal-notes.js',
```

Comment:
prettier did formatting, this is the actual mopdification in the file (path and added missing file)